src/text/Text.unit.tsx

```diff
@@ -16,7 +16,7 @@ describe('Text', () => {
   it('should have a role and classname', () => {
     const text = renderer
       .create(
-        <Text className="custom" role="role">
+        <Text className="custom" role="complementary">
```

Comment:
I know it's just a test but having this complementary  role on a text element is quite wrong and might only lead to people copy/pasting that bad exemple.
I would remove the role completely (why would you want to do that in the first place ?)

Reply:
I'm guessing it's to check that the prop `role` is passed, but it could be a unit test instead of a snapshot

Reply:
Yep and the snapshot to test custom class is useless too.
I'll check props another way